docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+
+```
+    var Stores = require('tux/Stores')
+```
+
+###<a id="Creating-Stores"></a>Creating Stores [#](#Creating-Stores)
+The `Stores` object allows the user to create stores via invoking [Stores.createStores](#createStores). Stores are a place in the Flux architecture to store and manipulate data. You can pass in the store methods that you have created to the `createStores` method and it will return an object that has been extended with the `emitChange`, `addChangeListener`, and `removeChangeListener` methods that are common to Flux stores. The Tux methods are added to the object being extended first, so if you want to overwrite Tux's implementation of one or all of those methods, you just need to provide your methods with those keys on the input object to `createStore`  Let's take a look at an example: [#](#create-store-example)
+<a id="create-store-example"></a>
+
+```
+    var messageStore = Stores.createStore({
+      _messages: {},
+      getAll: function () {
+        return Object.keys(this._messages);
+      }
+    });
+```
+
+Below is the API documentation for this method.
+
+###<a id="Stores-createStore"></a>Stores.createStore [#](#Stores-createStore)
+
+```
+    var store = Stores.createStore(methods);
+```
+
+####<a id="methods"></a>Parameter - `methods` - type: OBJECT - optional [#](#methods)
+Methods and properties to be extended on the store to be created. This object is optional and there are no expected keys.
+
+####<a id="storeReturn"></a>Return - `store` - type: OBJECT [#](#storeReturn)
+The `Stores.createStore` method returns an object which is described below.
+
+###<a id="store-emitChange"></a>Method `store.emitChange` - type: FUNCTION [#](#store-emitChange)
+A store instance is able to invoke the `emitChange` method, which will emit a change event string for any listeners that are listening for that particular string. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.emitChange(); // will emit the default change event type string: 'CHANGE'
+```
+
+#####<a id="emitChange-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#emitChange-CHANGE_EVENT)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.emitChange(newChangeEvent);
+```
+
+###<a id="store-addChangeListener"></a>Method `store.addChangeListener` - type: FUNCTION [#](#store-addChangeListener)
+A store instance is able to invoke the `addChangeListener` method, which will invoke the provided callback function when the specified event type string is emitted. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.addChangeListener(callback, CHANGE_EVENT);
```

Comment:
Add `//OR` to the end of this line and then add a line `store.addChangeListener(callback) //this will add a listener for the default CHANGE event`.